cmd/sops/main.go

```diff
@@ -1051,11 +1055,35 @@ func keyservices(c *cli.Context) (svcs []keyservice.KeyServiceClient) {
 }
 
 func inputStore(context *cli.Context, path string) common.Store {
-	return common.DefaultStoreForPathOrFormat(path, context.String("input-type"))
+	var configPath string
+	if context.String("config") != "" {
+		configPath = context.String("config")
+	} else {
+		// Ignore config not found errors returned from FindConfigFile since the config file is not mandatory
+		configPath, _ = config.FindConfigFile(".")
+	}
+	storesConf, _ := config.LoadStoresConfig(configPath)
+
+	return common.DefaultStoreForPathOrFormat(storesConf, path, context.String("input-type"))
 }
 
 func outputStore(context *cli.Context, path string) common.Store {
-	return common.DefaultStoreForPathOrFormat(path, context.String("output-type"))
+	var configPath string
+	if context.String("config") != "" {
+		configPath = context.String("config")
+	} else {
+		// Ignore config not found errors returned from FindConfigFile since the config file is not mandatory
+		configPath, _ = config.FindConfigFile(".")
+	}
+	storesConf, _ := config.LoadStoresConfig(configPath)
+<<<<<<< Updated upstream
+=======
+	if context.Int("indent") != 0 {
+		storesConf.YAML.Indent = context.Int("indent")
+	}
+>>>>>>> Stashed changes
```

Comment:
Merge issue

Reply:
Ho thanks, I forgot to save my file before committing. Fixed !